ppdet/data/transform/batch_operator.py

```diff
@@ -0,0 +1,555 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+try:
+    from collections.abc import Sequence
+except Exception:
+    from collections import Sequence
+
+import logging
+import cv2
+import numpy as np
+from .operator import register_op, BaseOperator, ResizeOp
+from .op_helper import jaccard_overlap, gaussian2D
+
+logger = logging.getLogger(__name__)
+
+__all__ = [
+    'PadBatchOp',
```

Comment:
PadBatchOp->PadBatch? the same as below

Reply:
use PadBatchOp as name to avoid conflicts because there is also a function named PadBatch in batch_operators